mysql-test/suite/data_masking/r/error_not_loaded_module.result

```diff
@@ -0,0 +1,3 @@
+CREATE FUNCTION gen_blacklist RETURNS STRING SONAME 'data_masking.so';
+select gen_blacklist();
+ERROR HY000: Can't initialize function 'gen_blacklist'; This function requires data_masking plugin which is not installed. Please instal
```

Comment:
is the error message truncated by mtr framework?

Reply:
No, it is not. It is cut off by the framework in https://github.com/ldonoso/percona-server/blob/32278aa683149716fdc5119939899b08de1d0851/mysys/my_error.c#L213

It uses the following format:

```
(gdb) p format                                                                                                                                                                                                                                         
+p format                                                                                                                                                                                                                                              
$9 = 0x555558375abd "Can't initialize function '%-.192s'; %-.80s
```

So the effective msg is effectively 80 chars long

Reply:
Looks good @ldonoso